annotation/src/main/java/com/google/errorprone/BugPattern.java

```diff
@@ -16,12 +16,12 @@
 
 package com.google.errorprone;
 
-import static java.lang.annotation.RetentionPolicy.RUNTIME;
-
 import java.lang.annotation.Annotation;
 import java.lang.annotation.Retention;
 import java.util.Comparator;
 
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
```

Comment:
Can you revert this import ordering change?  Google style is for static imports to go together at the top:
https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.3.3-import-ordering-and-spacing